styled-slider-view/src/main/java/co/sodalabs/view/StyledCompatSliderView.kt

```diff
@@ -0,0 +1,55 @@
+package co.sodalabs.view
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.support.v4.content.ContextCompat
+import android.support.v7.widget.AppCompatSeekBar
+import android.util.AttributeSet
+
+
+/**
+ * This is a wrapper of [AppCompatSeekBar] which uses "android:seekBarStyle"
+ * style attribute by default and this enable us the hot-theme function.
+ */
+class StyledCompatSliderView : AppCompatSeekBar {
+
+    companion object {
+        private const val MINIMUM_PROGRESS = 5
+    }
+
+    private val paint = Paint()
+    private val thumbBitmap by lazy { context.getBitmapFromVectorDrawable(R.drawable.ic_slider_thumb) }
```

Comment:
If we open source this, wouldn't you want the user to be able to set this via `styleable`?